### ISSUE TITLE
[Test Case] - Verify if function triggered by Cloud Event receives required headers

### ISSUE THREAD

**Issue by tgorgol** [kyma-project/kyma#9708]
**Summary**

Validates if the lambda function that was triggered by a Cloud Event receives all required Cloud Event headers.

**Prerequisites**
 - access to kyma runtime
 - namespace with [commerce mock](https://kyma-project.io/docs/1.16/root/getting-started/#getting-started-connect-an-external-application) connected, with Events API registered and instantiated in the namespace
 
**Environment**
Kyma runtime

**Test Steps**
- log in to console UI and select a namespace
- create functions for Node.js 10 and Node.js 12 runtimes with handler:
```js
module.exports = { 
  main: function (event, context) {
    console.log(event);
  }
}
```
- create a function for Python 3.8 runtime with handler:
```py
def main(event, context):
    print(event)
```
- add Event Trigger to each created function, listening for `customer.created` events 
- use commerce mock to emit `customer.created` event
- check each function's logs to verify if the `event` object was correctly populated

**Expected result**
The logged `event` object has properties corresponding to Cloud Event headers (starting with `ce-*`, e.g. `ce-type`) correctly populated (not `undefined` or `None`).

**Comment by pbochynski** [comment]
Test for node.js function is implemented in #10259

**Comment by kwiatekus** [comment]
It is tested already in an automated way
